Observer/src/observer/Channel.java

```diff
@@ -0,0 +1,31 @@
+package observer;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Channel {
```

Comment:
There is no abstraction for channel,
SOLI **D**

Reply:
Ya. I forgot here that I am not following Dependency Injection.